Tests/Adyen Tests/Analytics/AnalyticsProviderMock.swift

```diff
@@ -26,4 +26,19 @@ class AnalyticsProviderMock: AnalyticsProviderProtocol {
     var initialEventCalled: Bool {
         initialEventCallsCount > 0
     }
+    
+    var infoCount = 0
+    func add(info: Adyen.AnalyticsEventInfo) {
+        infoCount += 1
```

Comment:
I think for the future tests for SyncDataStore it could be beneficial to capture real incoming events and make count to be a calculated var, but this is something for the next batch